Source/FORMData.m

```diff
@@ -875,11 +875,13 @@ - (NSArray *)updateTargets:(NSArray *)targets
 
 - (NSArray *)enableTargets:(NSArray *)targets
 {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips object:[NSNumber numberWithBool:YES]];
     return [self updateTargets:targets withEnabled:YES];
 }
 
 - (NSArray *)disableTargets:(NSArray *)targets
 {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips object:[NSNumber numberWithBool:NO]];
```

Comment:
Do you mind using a literals here? 

`@NO` => `[NSNumber numberWithBool:NO]`

Method formatting `:` goes in newlines
```objc
[[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips
                                                    object:@NO];
```